data/sheragi/archaeology missions.txt

```diff
@@ -345,17 +345,18 @@ mission "Sad Archie"
 		has "There Might Be Riots part 3B: done"
 	destination "Zug"
 	
-	npc disable
-		government "Test Dummy"
-		system "Ildaria"
-		personality unconstrained heroic nemesis
-		ship "Timer Ship" "Timer"
-		conversation
-			`As your ship drifts through the cloud of dust that surrounds Ildaria, it occurs to you that this must be the "cloudy star out along the Rim" that Ulrich from There Might Be Riots told you about. Remembering his story, you shut your eyes and attempt to meditate. But instead, you just fall asleep.`
-			scene "scene/eso0"
-			`You dream that you leave this star system and return to Zug, but when you land there everything is different: the cities you remember are gone, and in their place are other cities with strange architecture, populated by gargantuan dragon-like creatures.`
-			`	Time speeds up, and you watch as the dragon cities grow and spread out. You see one city buried under a massive lava flow, starships buzzing around it like a cloud of flies to carry people and goods to safety. Other cities are destroyed by war, and then abruptly the dragon-people are all gone and their cities lie vacant. Buildings crumble, and the forests reclaim the land.`
-			`	Then you wake up and find yourself back in your ship, in the Ildaria system. Weird.`
+	timer "Sad Archie Timer" 120 60
+		system Ildaria
```

Comment:
Should this be a location filter instead?

Reply:
For this specific mission, definitely not; it's explicitly supposed to be Ildaria.

In general...I considered it, but it seems to me that if you're giving a player an invisible timer, it's at least polite to make sure they know _exactly_ where they have to wait for it, rather than expecting them to guess one of N systems.

I would think that, if it _were_ to support location filters, there would need to be a way to get the mission text to recognize which system was selected, so it could pass that information along. This is particularly tricky if a mission has more than one timer (which is supported right now). Either that, or the location filter would have to be on the mission itself, and there would have to be a way of passing that information through to the timer.

Unless I'm missing something, which is entirely possible.

Reply:
Oh, huh. I hadn't known that was how news worked (though it makes sense)—I was only thinking of location filters in terms of picking a specific system from among them.

That doesn't seem too difficult a change.